src/main/resources/META-INF/plugin.xml

```diff
@@ -30,12 +30,39 @@
     <!-- Description of the plugin displayed on the Plugin Page and IDE Plugin Manager.
          Simple HTML elements (text formatting, paragraphs, and lists) can be added inside of <![CDATA[ ]]> tag.
          Guidelines: https://plugins.jetbrains.com/docs/marketplace/plugin-overview-page.html#plugin-description -->
-    <description><![CDATA[
-      TornadoInsight is an open source IntelliJ plugin that aims to enhance the experience of developers when working with TornadoVM. 
-      It provides  a built-in on-the-fly static checker, empowering developers to identify unsupported Java features in TornadoVM and understand the reasons behind these limitations. 
-      Additionally, it introduces a dynamic testing framework that enables developers to easily test individual TornadoVM tasks. 
-      It automatically wraps TornadoVM tasks, invoking the native TornadoVM runtime on the developers' machines for seamless debugging and testing. 
-  ]]></description>
+    <description><![CDATA[TornadoInsight is an open source IntelliJ plugin that aims to enhance the experience of developers when working with
+<a href="https://github.com/beehive-lab/TornadoVM">TornadoVM</a>.
+<ul>
+    <li>TornadoInsight provides a built-in on-the-fly static checker, empowering developers to identify unsupported
+        Java features in TornadoVM and understand the reasons behind these limitations.</li>
+    <li>TornadoInsight introduces a dynamic testing framework that enables developers to easily test individual TornadoVM tasks.
+        It automatically wraps TornadoVM tasks, invoking the native TornadoVM runtime on the developers' machines for seamless
+        debugging and testing.</li>
+</ul>
+<br/>
+<h2>Getting started</h2>
+
+TornadoInsight is available for versions 2022.3 and higher of compatible JetBrains IDEs.
+It invokes at runtime Java and TornadoVM on the developers' local machine, and therefore,
+developers need to make sure that they have installed those prior to using the plugin:
+<ul>
+    <li><a href="https://github.com/beehive-lab/TornadoVM">TornadoVM</a> >= 1.0</li>
+    <li>JDK >= 21</li>
+</ul>
+
+The dynamic inspection feature of TornadoInsight is configured after the installation of the plugin, as follows:
+<ul>
+    <li>Navigate to "Preferences" or "Settings" depending on your operating system.</li>
```

Comment:
In both Ubuntu and macOS, I have seen "Settings". Is there any particular OS that you have seen "Preferences"?

Is it OS-specific? or does it depend to the IntelliJ version?

Reply:
I remember IDEA using “Preference” in macOS for version 2021 and before, but they seem to use "Settings" since then, and since TornadoInsight is compatible with versions after 2022.1, I think it's fine to just keep "Settings"